org.metaborg.spoofax.shell.eclipse/src/main/java/org/metaborg/spoofax/shell/client/eclipse/impl/hooks/EclipseMessageHook.java

```diff
@@ -0,0 +1,31 @@
+package org.metaborg.spoofax.shell.client.eclipse.impl.hooks;
+
+import org.metaborg.spoofax.shell.client.IDisplay;
+import org.metaborg.spoofax.shell.client.hooks.IMessageHook;
+import org.metaborg.spoofax.shell.output.StyledText;
+
+import com.google.inject.Inject;
+
+/**
+ * An Eclipse-based implementation of {@link IMessageHook}.
+ */
+public class EclipseMessageHook implements IMessageHook {
```

Comment:
The two hooks implemented here are identical to the implementations in the console module; perhaps these should be moved to the core module in one way or another.